apps/aecore/src/aec_peers.erl

```diff
@@ -60,27 +60,32 @@
 
 %% We parse the uri's with http_uri, therefore we use types from that module.
 -record(peer, {
-          uri = ""          :: http_uri:uri(),   %% try not to use!
+          uri = ""          :: http_uri_uri(),   %% try not to use!
           scheme            :: http_uri:scheme(),
-          host              :: http_uri:host(),
-          port              :: http_uri:port(),
-          path              :: http_uri:path(),
+          host              :: http_uri_host(),
+          port              :: http_uri_port(),
+          path              :: http_uri_path(),
```

Comment:
You need `http_uri_path() | 'undefined'` here, or adapt init/0 to set this field.

Reply:
... or delete path altogether as it seems unused.